src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.figure;
+
+public interface Figure {
+    double getArea();
+
```

Comment:
interfaces should be named and describe behavior like `Drawable`, `Cloneable` `Serializable`
\+ redundant empty line

Reply:
I made changes.